### PR TITLE
news: update v0.19.0 news to included missed fixes

### PR DESCRIPTION
**Pull request by ajeddeloh**
Two bug fixes were omitted from the v0.19.0 release notes by mistake. Add them to the v0.19.0 NEWS entry.

### CODE REVIEW

File: NEWS [hunk 1]
```diff
@@ -18,6 +18,10 @@
       reported correctly
     - Add warning if a non-existent filesystem is specified when creating links
       and directories
+    - Fix udev race causing units depending on Ignition and device to fail when
+      no filesystems are created
+    - Fix udev race where symlinks are deleted before Ignition can create its own
```
Comment (bgilbert) [inline NEWS]: nit: trailing period

File: NEWS [hunk 2]
```diff
@@ -18,6 +18,10 @@
       reported correctly
     - Add warning if a non-existent filesystem is specified when creating links
       and directories
+    - Fix udev race causing units depending on Ignition and device to fail when
```
Comment (bgilbert) [inline NEWS]: `depending on Ignition and device` seems unclear.

**Comment by ajeddeloh** [comment]
Made more specific, fixed trailing period.

File: NEWS [hunk 3]
```diff
@@ -18,6 +18,10 @@
       reported correctly
     - Add warning if a non-existent filesystem is specified when creating links
       and directories
+    - Fix udev race causing systemd units depending on the Ignition disk stage and
```
Comment (bgilbert) [inline NEWS]: s/disk/disks/

**Pull request merged**